### PR TITLE
Feature/oscbs 1 bacnet sc implementation mongoose

### PR DESCRIPTION
**Pull request by Michail-Antropov**

### CODE REVIEW

File: ports/zephyr/websocket-cli.c [hunk 1]
```diff
@@ -66,7 +71,7 @@ typedef enum {
 } WORKER_ID_EVENT;
 
 #define CONTEXT_PORT_NO_LISTEN 0
-#define TLS_PEER_HOSTNAME "localhost"
+//#define TLS_PEER_HOSTNAME "localhost"
```
Comment (cybermag) [inline ports/zephyr/websocket-cli.c:74]: That commented code must be removed

File: zephyr/tests/ports/bsc_event/src/main.c [hunk 2]
```diff
@@ -24,12 +24,18 @@ static TEST_STAGE test_stage = STAGE_NONE;
 #define TIMEOUT_CHILD   400
 #define TIMEOUT_MIN     200
 #define TIMEOUT_MAX     600
+//#define MSEC_PER_SEC    1000
```
Comment (cybermag) [inline zephyr/tests/ports/bsc_event/src/main.c:27]: commented code must be removed

File: zephyr/tests/ports/bsc_event/prj.conf [hunk 3]
```diff
@@ -0,0 +1,12 @@
+CONFIG_ZTEST=y
+CONFIG_BACNETSTACK=y
+CONFIG_BACDL_BSC=y
+
+CONFIG_BACNETSTACK_LOG_LEVEL_ERR=y
```
Comment (cybermag) [inline zephyr/tests/ports/bsc_event/prj.conf:5]: Do we really need logging here?

File: zephyr/tests/bacnet/datalink/bsc-datalink/prj.conf [hunk 4]
```diff
@@ -0,0 +1,108 @@
+CONFIG_ZTEST=y
+CONFIG_BACNETSTACK=y
+CONFIG_HEAP_MEM_POOL_SIZE=100000
+CONFIG_MAIN_STACK_SIZE=20000
+CONFIG_INIT_STACKS=y
+
+# Debugging stuff (can be commented out)
+CONFIG_PRINTK=y
```
Comment (cybermag) [inline zephyr/tests/bacnet/datalink/bsc-datalink/prj.conf:8]: Review enabled logging

**Review by cybermag — CHANGES_REQUESTED** [review]

File: ports/zephyr/websocket-cli.c (hunk 1 above)
Comment (Michail-Antropov) [reply ports/zephyr/websocket-cli.c:74]: Done

File: zephyr/tests/ports/bsc_event/src/main.c (hunk 2 above)
Comment (Michail-Antropov) [reply zephyr/tests/ports/bsc_event/src/main.c:27]: Done

File: zephyr/tests/ports/bsc_event/prj.conf (hunk 3 above)
Comment (Michail-Antropov) [reply zephyr/tests/ports/bsc_event/prj.conf:5]: done

File: zephyr/tests/bacnet/datalink/websockets/prj.conf [hunk 5]
```diff
@@ -1,23 +1,68 @@
...
+CONFIG_MAIN_STACK_SIZE=20000
+CONFIG_INIT_STACKS=y
 
-# logging
-#CONFIG_NET_CONN_LOG_LEVEL_DEFAULT=n
-#CONFIG_NET_CONN_LOG_LEVEL_DBG=y
-#CONFIG_NET_IF_LOG_LEVEL_DEFAULT=n
-CONFIG_NET_IF_LOG_LEVEL_DBG=y
-CONFIG_NET_LOG=y
+# Debugging stuff (can be commented out)
```
[the remark this answers is not in the document]
Comment (Michail-Antropov) [reply zephyr/tests/bacnet/datalink/websockets/prj.conf:7]: done

File: zephyr/tests/bacnet/datalink/bvlc-sc/prj.conf [hunk 6]
```diff
@@ -0,0 +1,108 @@
+CONFIG_ZTEST=y
+CONFIG_BACNETSTACK=y
+CONFIG_HEAP_MEM_POOL_SIZE=80000
+CONFIG_MAIN_STACK_SIZE=20000
+CONFIG_INIT_STACKS=y
+
+# Debugging stuff (can be commented out)
+CONFIG_PRINTK=y
```
[the remark this answers is not in the document]
Comment (Michail-Antropov) [reply zephyr/tests/bacnet/datalink/bvlc-sc/prj.conf:8]: done

File: zephyr/tests/bacnet/datalink/bsc-socket/prj.conf [hunk 7]
```diff
@@ -0,0 +1,87 @@
+CONFIG_ZTEST=y
+CONFIG_BACNETSTACK=y
+CONFIG_HEAP_MEM_POOL_SIZE=80000
+CONFIG_MAIN_STACK_SIZE=20000
+CONFIG_INIT_STACKS=y
+
+# Debugging stuff (can be commented out)
```
[the remark this answers is not in the document]
Comment (Michail-Antropov) [reply zephyr/tests/bacnet/datalink/bsc-socket/prj.conf:7]: done

File: zephyr/tests/bacnet/datalink/bsc-datalink/prj.conf (hunk 4 above)
Comment (Michail-Antropov) [reply zephyr/tests/bacnet/datalink/bsc-datalink/prj.conf:8]: done

**Pull request merged**